### PR TITLE
updated docs for fetch

### PR DESCRIPTION
**Pull request by madhur-tandon**
Inspired by @JeffersGlass
Closes https://github.com/pyscript/pyscript/issues/1007

### CODE REVIEW

**Comment by madhur-tandon** [comment]
Should probably be merged after @FabioRosado's https://github.com/pyscript/pyscript/pull/982

**Review by marimeireles — COMMENTED** [review]
Thanks for the work here!

File: docs/reference/elements/py-config.md [hunk 1]
```diff
@@ -235,10 +235,187 @@ A fetch configuration consists of the following:
...
+- If `files` is specified
+  - for each `file` present in the `files` array
+    - the `sourcePath` is calculated as `from + file`
+    - the `destination` is calculated as `to_folder + file`
+      - thus, the object is downloaded from `sourcePath` to `destination`
+- Else i.e. `files` is NOT specified
+  - If `to_file` is specified
+    - the object is downloaded from `from` to `to_folder + to_file`
+  - Otherwise, calculate the `filename` at the end of `from` i.e. the part after last `/`
+    - the object is downloaded from `from` to `to_folder + filename at the end of 'from'`
```
[the remark this answers is not in the document]
Comment (madhur-tandon) [reply docs/reference/elements/py-config.md:258]: The blog is in draft mode right now and isn't published yet.

[the remark this answers is not in the document]
Comment (madhur-tandon) [reply docs/reference/elements/py-config.md:258]: So maybe we shouldn't link it right now, what say @marimeireles 
CC: @JeffersGlass -- unless you know the link of the production ready blog beforehand?

**Review by FabioRosado — APPROVED** [review]

**Pull request merged**